public/app/core/directives/misc.js

```diff
@@ -18,6 +18,17 @@ function (angular, coreModule, kbn) {
     };
   });
 
+  coreModule.default.directive('dtip', function($compile) {
```

Comment:
If I have an AngularJS variable like "panel.helpText", I am not able display it inside a tooltip because in "tip" directive we add quotes. But in the above dtip (dynamic tip) directive, we can get the value of the AngularJS variable by using it like the following:
<dtip>panel.helpText</dtip>